apps/user-cdc/src/main/java/it/pagopa/selfcare/user/event/config/TelemetryClientConfig.java

```diff
@@ -0,0 +1,21 @@
+package it.pagopa.selfcare.user.event.config;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.TelemetryConfiguration;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
+public class TelemetryClientConfig {
+
+    @ConfigProperty(name = "user-cdc.appinsights.connection-string")
+    String appInsightsConnectionString;
+
+    @ApplicationScoped
+    public TelemetryClient telemetryClient() {
```

Comment:
if `appInsightsConnectionString` is only used for construct telemetryClient you could inject it as follow:

```
public TelemetryClient telemetryClient(@ConfigProperty(name = "user-cdc.appinsights.connection-string") String appInsightsConnectionString) {
....
```